### PR TITLE
Add art files and build script

### PR DESCRIPTION
**Pull request by floogulinc**

### CODE REVIEW

File: build-art.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+  }
+  console.log(`Done Sharp processing ${dirPath} to ${outputDirPath}`);
+  await imagemin([`${outputDirPath}/*.png`], {
+    destination: outputDirPath,
+    plugins: [
+        imageminPngquant({
+            quality: [0.3, 0.8]
+        })
+    ]
```
Comment (helloworld12321) [inline build-art.js:44]:
```suggestion
    plugins: [
      imageminPngquant({
        quality: [0.3, 0.8]
      })
    ]
```
Just a quick indentation thing here

File: build-art.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const sharp = require('sharp');
+
+console.log('Sharp Versions:');
+console.log(sharp.versions);
+console.log(`Sharp Concurrency: ${sharp.concurrency()}`);
+
```
Comment (helloworld12321) [inline build-art.js:7]:
```suggestion
```

Is there a reason to keep these `console.log()`s? (If there is, never mind then :slightly_smiling_face:)

Comment (floogulinc) [reply build-art.js:7]: I felt like they are useful to have in the logs

**Comment by helloworld12321** [comment]
I've already taken a pretty thorough looked at this, and we know everything works, so I'll just approve it right away :)

**Pull request merged**